inst/make_cmdstan.sh

```diff
@@ -4,14 +4,17 @@
 #  - build binaries, compile example model to build model header
 #  - symlink downloaded version as "cmdstan"
 
-while getopts ":d:v:j:" opt; do
+WIN=0
+while getopts ":d:v:j:w" opt; do
```

Comment:
The changes to this script look good. I assume it does, but if you haven’t already can you check that it still works when _not_ on windows?

Reply:
Actually never mind, don’t worry about that. Travis will do that for us.

Reply:
I was counting on Travis yes, though I did go in and check now and its all good.